src/main/java/org/gridsuite/securityanalysis/server/SecurityAnalysisController.java

```diff
@@ -118,4 +118,21 @@ private static SecurityAnalysisParameters getNonNullParameters(SecurityAnalysisP
         Mono<Void> result = service.deleteResults();
         return ResponseEntity.ok().body(result);
     }
+
+    @GetMapping(value = "/status/{resultUuid}", produces = APPLICATION_JSON_VALUE)
+    @Operation(summary = "Get the security analysis status from the database")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The security analysis status")})
+    public ResponseEntity<Mono<String>> getStatus(@Parameter(description = "Result UUID") @PathVariable("resultUuid") UUID resultUuid) {
+        Mono<String> result = service.getStatus(resultUuid);
+        return ResponseEntity.ok().body(result);
+    }
+
+    @PutMapping(value = "/status/{resultUuid}", produces = APPLICATION_JSON_VALUE)
```

Comment:
To be consistent with the other /results/{resultUuid}/status ?

Reply:
Corrected